docs/rst/user_manual/configuration.rst

```diff
@@ -685,6 +685,93 @@ It will use such profile for configuring the Participant.
 
 .. _user_manual_configuration_general_example:
 
+
+Forwarding Routes
+=================
+
+The |ddsrouter| is capable of establishing different internal routes between its participants.
+This feature enables users to only send sensible data to a set of participants.
+
+.. note::
+
+    By default, when the tag ``routes`` is not set, every participant receives everything.
+
+Generic Routes
+--------------
+
+To configure a custom set of forwarding routes, use the tag ``routes`` followed by the destination participants of each source.
+
+
+.. note::
+
+    If a participant is not listed as a source, it will send the data it receives to every participant.
+
+.. note::
+
+    If a participant is listed as a source but it is not given any destination participants, it will not send the data it receives to any participant.
+
+
+Consider the following example with three participants: ``Participant0``, ``Participant1``, and ``Participant2``.
+
+.. code-block:: yaml
+
+  routes:
+    - src: Participant0
+      dst:
+        - Participant2
+
+    - src: Participant1
+
+
+When data is published:
+
+* Participant ``Participant0`` will only forward the data it receives to participant ``Participant2``.
+* Participant ``Participant1`` will not forward the data it receives to any participant, since it does not have any destination participants.
+* Participant ``Participant2`` will forward the data it receives to every participant, since it does not have a forwarding route.
```

Comment:
```suggestion
* Participant ``Participant2`` will forward the data it receives to every participant (``Participant0`` and ``Participant1``), since it does not have a forwarding route.
```